api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ResourceMonitoringManager.java

```diff
@@ -243,15 +264,21 @@ private void processIdleRuns(final List<PipelineRun> runs) {
                 double cpuUsageRate = metric / MILLIS / type.getVCPU();
                 if (Precision.compareTo(cpuUsageRate, idleCpuLevel, ONE_THOUSANDTH) < 0) {
                     processIdleRun(run, actionTimeout, action, runsToNotify, runsToUpdate, cpuUsageRate);
+                    if (run.addTag(UTILIZATION_LEVEL_LOW, TRUE_VALUE_STRING)) {
+                        runsToUpdateTags.add(run);
```

Comment:
actually can we use runsToUpdate to update tags as well? Am I right that it will contain all runs with updated tags? and we can just do:
```
if (Precision.compareTo(cpuUsageRate, idleCpuLevel, ONE_THOUSANDTH) < 0) {
    processIdleRun(run, actionTimeout, action, runsToNotify, runsToUpdate, cpuUsageRate);
    run.addTag(UTILIZATION_LEVEL_LOW, TRUE_VALUE_STRING)
} else if (run.getLastIdleNotificationTime() != null) {
    run.setLastIdleNotificationTime(null);
    run.removeTag(UTILIZATION_LEVEL_LOW, TRUE_VALUE_STRING)
    runsToUpdate.add(run);
}
...
pipelineRunManager.updatePipelineRunsLastNotification(runsToUpdate);
pipelineRunManager.updateRunsTags(runsToUpdate);
```